src/ranged.cpp

```diff
@@ -1851,21 +1841,38 @@ item::sound_data item::gun_noise( const bool burst ) const
 
     noise = std::max( noise, 0 );
 
-    if( ammo_current() == itype_40x46mm || ammo_current() == itype_40x53mm ) {
+    if( has_flag( flag_SOUND_LAUNCHER ) ) {
         // Grenade launchers
-        return { 8, _( "Thunk!" ) };
+        return { noise, _( "Thunk!  Boom!" ) };
```

Comment:
Given discussion and testing in the discord, we can try and fix frag ammo not making any noise in a later PR I figure, so this probably might work well enough for now. Also went with something a lil more different than the crossbow noise, if desired.
```suggestion
        return { noise, _( "Thump!" ) };
```